src/components/Room/Room.tsx

```diff
@@ -96,9 +113,24 @@ export function Room({
   }
 
   receiveMessage(message => {
+    const userSettings = settingsContext.getUserSettings()
+    !shellContext.tabHasFocus &&
+      userSettings.playSoundOnNewMessage &&
+      playNewMessageSound()
     setMessageLog([...messageLog, { ...message, timeReceived: Date.now() }])
   })
 
+  const playNewMessageSound = () => {
+    if (!audioBufferContainer.current) {
+      console.error("Can't fetch sound buffer to play")
+      return
+    }
+    const audioSource = audioContext.createBufferSource()
+    audioSource.buffer = audioBufferContainer.current
+    audioSource.connect(audioContext.destination)
```

Comment:
Does this need to run each time the audio is played? If not, could it be moved to where the audio file is fetched, so that it only runs once?

Reply:
From what i found on internet, from multiple sources, it seems so. 

https://web.dev/webaudio-intro/
https://stackoverflow.com/a/12327320

Reply:
Got it. Thank you for getting to the bottom of this!